packages/editor-ui/src/Interface.ts

```diff
@@ -316,6 +317,36 @@ export interface IWorkflowDb {
 	meta?: WorkflowMetadata;
 }
 
+// For workflow list we don't need the full workflow data
+export type BaseResource = {
+	id: string;
+	name: string;
+};
+
+export type WorkflowListItem = Omit<
+	IWorkflowDb,
+	'nodes' | 'connections' | 'settings' | 'pinData' | 'versionId' | 'usedCredentials' | 'meta'
+> & {
+	resource: 'workflow';
+	parentFolder?: { id: string; name: string };
+};
+
+export type BaseFolderItem = BaseResource & {
+	createdAt: string | number;
+	updatedAt: string | number;
```

Comment:
Any reason this can be string or number?

Reply:
It's like that in the `IWorkflowDB` but I don't think we have a case where it's a number. At least it's coming as string from API, so will update